css/style.css

```diff
@@ -0,0 +1,92 @@
+
+/*#fadeImage{
+  background: white;
+}*/
+
+
+#container{
+  display: none;
+  max-width: 960px;
+  margin: auto;
+}
+
+body{
+  background: url("space4.gif");
+}
+
+#page-header{
+  border:0;
+  box-sizing: border-box;
+  clear: both;
+  display: block;
+  float: left;
+  margin: auto;
+  padding: 2% 8% 2% 8%;
+  width: 100%;
+}
+
+.clearfix{
+  clear: both;
+}
+
+
+.nav li{
+  border: 2px white solid;
+  display: inline-block;
+  float: right;
+  margin: 2%;
+  padding: 1%;
+}
+
+.nav li:hover{
+  background: white;
+}
+
+.nav li a:hover{
+  color: black;
+}
+
+.nav li a{
+  color: white;
+  font-size: 24px;
+  list-style: none;
+  text-decoration: none;
+  width: 15%;
+}
+
+main{
+  box-sizing: border-box;
```

Comment:
Rather than applying this to a single element, you should apply the same `box-sizing` to all elements on your page so they behave consistently. 

```
* {
 box-sizing: border-box;
}
```